example-schema/type-hierarchy.yaml

```diff
@@ -2,11 +2,18 @@ test:
   label: Test
   description: Test types
   types:
-    - MainType
-    - ParentType
-    - ChildType
     - RestrictedType
     - OddCodeType
     - MVRType
+    - MVRTypeChild
     - FieldsetType
     - ValidatedType
+    - PropertiesTest
```

Comment:
Highlight #4 there are now more types more directly tailored to testing specific things. Still not a perfect separation of concerns, but a big step in the right direction